src/main/java/io/github/darealturtywurty/ancientology/core/data/ItemModelGenerator.java

```diff
@@ -17,8 +19,9 @@ public ItemModelGenerator(DataGenerator generator, ExistingFileHelper existingFi
 
     @Override
     protected void registerModels() {
-        // defaultBlock(name, blockitem);
-        // defaultItem(name, item);
+        defaultBlock(BlockInit.DEEPSLATE_TIN_ORE.getId(), (BlockItem) BlockInit.DEEPSLATE_TIN_ORE.get().asItem());
+        defaultItem(ItemInit.RAW_TIN.getId(), ItemInit.RAW_TIN.get());
+        defaultItem(ItemInit.TIN_INGOT.getId(), ItemInit.TIN_INGOT.get());
```

Comment:
Couldn't you make the `defaultItem` and `defaultBlock` methods take in `ItemProvider` respectively `BlockProvider` so you don't need `#get`s?
Side note: `BlockRegistryObject` implements `ItemProvider` (and `ItemLike`) which means that you don't need to add `.get().asItem()`.
```suggestion
        defaultBlock(BlockInit.DEEPSLATE_TIN_ORE, BlockInit.DEEPSLATE_TIN_ORE);
        defaultItem(ItemInit.RAW_TIN, ItemInit.RAW_TIN);
        defaultItem(ItemInit.TIN_INGOT, ItemInit.TIN_INGOT);
```

Reply:
I didn't make the `defaultBlock` & `defaultItem` and didn't want to change them, I did ask in the discord however no one said anything so I left it.